spiceaidocs/content/en/reference/Secrets/_index.md

```diff
@@ -0,0 +1,119 @@
+---
+type: docs
+title: "Spice Secrets"
+linkTitle: "Spice Secrets"
+description: ''
+weight: 70
+---
+
+```mermaid
+
+block-beta
+    columns 5
+
+    store["secret store"] space secret_1["spiceai secret"] space v1["'key'"]
+    space space space space space
+    space space secret_2["dremio secret"] space v3["'user'"]
+    space space space space v4["'password'"]
+
+    store -- "spiceai" --> secret_1
+    store -- "dremio" --> secret_2
+
+    secret_1 -- "key" --> v1
+    secret_2 -- "user" --> v3
+    secret_2 -- "password" --> v4
+
+```
+
+## Secret
+
+A Secret is an object used to store sensitive data, like a passwords, a tokens, or keys.
+
+## Secret Stores
+
+The secret store is a location where the `secret` objects are stored.
+
+Spice.ai supports four types of secret stores: `file`, `env`, `kubernetes` and `keyring`. The type of secret store is specified in the `store` field of the `secrets` section in the Spicepod manifest.
+
+### File Secret Store
+
+
+The Spice.ai CLI provides two commands, `spice login` and `spice login dremio`, which are designed to simplify the process of storing credentials. When executed, these commands will automatically persist the corresponding credentials in the file secret store.
```

Comment:
Remove Dremio specific information (that would be provided by the Dremio extension docs).

Reply:
Removed